src/RA_Achievement.h

```diff
@@ -52,7 +73,8 @@ class Achievement
     inline BOOL GetPauseOnReset() const { return m_bPauseOnReset; }
     void SetPauseOnReset(BOOL bPause) { m_bPauseOnReset = bPause; }
 
-    BOOL IsCoreAchievement() const { return m_nSetType == Core; }
+    _Success_(return)
+        _NODISCARD _CONSTANT_FN IsCoreAchievement() const noexcept { return (m_nSetType == Type::Core); }
```

Comment:
Figured out a way to not need to get ride of this.

Reply:
I'd like you to try to get rid of it. It's only used for activating the achievement in `Parse`, which overloads the functionality of `Parse` - `Parse` should _just_ parse the string. Activiation should happen outside of `Parse`, where we already know if it's a Core achievement or not.

Reply:
Accidently hit resolved here on my phone instead of reply. But sure it should be easy enough.